pyproject.toml

```diff
@@ -0,0 +1,104 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools>=61.2", "wheel"]
+
+[project]
```

Comment:
do we not need a version number like we had in setup.py (imported from another file)

Reply:
This is now handled with
```
[tool.setuptools.dynamic.version]
attr = "xgi.__version__"
```

Reply:
Thanks I missed that